Test/Utils/ContrailAPI/DNSServerRepo_Impl.Tests.ps1

```diff
@@ -0,0 +1,114 @@
+Param (
+    [Parameter(Mandatory=$false)] [string] $TestenvConfFile,
+    [Parameter(ValueFromRemainingArguments=$true)] $UnusedParams
+)
+
+. $PSScriptRoot\DNSServerRepo.ps1
+. $PSScriptRoot\IPAMRepo.ps1
+. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testenv.ps1
+
+# TODO: Those tests run on working Controller.
+#       Most probably they need to be rewrote
+#       to use some fake.
+Describe 'Configure DNS Class API' -Tags CI, Systest {
+    BeforeAll {
+        $OpenStackConfig = Read-OpenStackConfig -Path $TestenvConfFile
+        $ControllerConfig = Read-ControllerConfig -Path $TestenvConfFile
+        $ContrailNM = [ContrailNetworkManager]::New($OpenStackConfig, $ControllerConfig)
+
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute(
+            "PSUseDeclaredVarsMoreThanAssignments",
+            "DNSServer",
+            Justification="It's actually used."
+        )]
+        $DNSServerRepo = [DNSServerRepo]::New($ContrailNM)
+    }
+
+    Context 'DNS server creation and removal' {
+
+        It 'can add and remove not attached to ipam DNS server without records' {
+            {
+                $Server = [DNSServer]::New("CreatedByPS1ScriptEmpty")
+                $DNSServerRepo.AddContrailDNSServer($Server)
+                $DNSServerRepo.RemoveContrailDNSServer($Server, $false)
+            } | Should -Not -Throw
+        }
+
+        It 'can remove DNS server by name' {
```

Comment:
I cannot see the difference between this test case and `can add and remove not attached to ipam DNS server without records` test case and in my opinion they can be merged into one. Could you elaborate what is the difference?

Reply:
Important information: while creating server, field Uuid of server object is filled.
In first test, removing function uses the uuid set while creating. In second test, before removing, it asks contrail for uuid of server with specific FQName.